meta-facebook/mfz-wc/src/platform/plat_sensor_table.h

```diff
@@ -83,6 +84,7 @@
 #define SENSOR_NUM_CUR_PVCCFA_EHV 0x33
 #define SENSOR_NUM_CUR_PVCCD_HV 0x34
 #define SENSOR_NUM_CUR_PVCCINFAON 0x35
+#define SENSOR_NUM_CUR_IOM_P12V 0xFF // Reserved the macro temporarily
```

Comment:
should not be 0xFF, cause 0xFF is invalid sensor number for lots of sensor relative api, just increase sensor number after latest one temporary.

Reply:
Done.